### ISSUE TITLE
Add name:ml to places

### ISSUE THREAD

**Issue by naveenpf** [osmkerala/osmkerala-tasks#10]
**To add ml tags (example)**
Run the overpass query :- http://overpass-turbo.eu/s/eNn (change the name:ml to your language)
Click on export and select Level0 editor .
Add name:ml tag for each of entity.
Add the change comment and click of upload to OSM

**Comment by naveenpf** [comment]
https://www.openstreetmap.org/user/mdmahir/diary/46883

Check whether https://www.varnamproject.com/ can be used

**Comment by naveenpf** [comment]
@subins2000 @navaneeth @joicemjoseph

**Comment by joicemjoseph** [comment]
Will check

**Comment by subins2000** [comment]
Yes, varnam is a good method. But for some names the suggestion might not be correct.

**Comment by joicemjoseph** [comment]
https://github.com/libindic/indic-trans

**Comment by subins2000** [comment]
A request to this API can retrieve the Malayalam transliteration : https://api.varnamproject.com/tl/ml/Kunnamkulam

**Comment by naveenpf** [comment]
csv file:- http://overpass-turbo.eu/s/YSc

**Comment by subins2000** [comment]
Just to confirm, this still hasn't been done right ?

**Comment by subins2000** [comment]
Made a script to add a new column to csv with transliteration. All in this zip :

[name-ml.zip](https://github.com/osmkerala/osmkerala-tasks/files/5895885/name-ml.zip)

**Comment by naveenpf** [comment]
this application looks promising https://osmnames.netlify.app/
